src/movies-servies.ts

```diff
@@ -0,0 +1,129 @@
+import logger from "./logger";
+import { sortByField, paginate } from "./utils";
+import { getOMDBMovie } from "./omdb-service";
+
+const moviesDB: any = {};
+
+export const addMovie = async (req: any, res: any) => {
+  logger.info({
+    request_id: req.id,
+    url: req.url,
+    body: req.body,
+    message: "add movie",
+  });
+
+  const title: string = req.body.title;
+
+  if (!title) {
+    return res.send({ error: "Title is required!" });
+  }
+  const { status, data }: any = await getOMDBMovie(title);
+  const id: number = new Date().getUTCMilliseconds();
```

Comment:
It is unsafe decision, you are limited with 1000 IDs and have a chance to create two or more movies with the same ID.